apps/explorer/src/ui/stories/DisclosureBox.stories.tsx

```diff
@@ -16,5 +16,9 @@ export const DisclosureBoxDefault: StoryObj<DisclosureBoxProps> = {
 
 export const DisclosureBoxClosed: StoryObj<DisclosureBoxProps> = {
     ...DisclosureBoxDefault,
-    args: { title: 'Expanded disclosure box', defaultOpen: true },
+    args: {
```

Comment:
Can you add an `outline` variant example?

Reply:
This is done